c10/util/BFloat16-inl.h

```diff
@@ -15,3 +16,21 @@ inline C10_HOST_DEVICE BFloat16::operator float() const {
 }
 
 } // namespace c10
+
+namespace std {
+
+template <>
```

Comment:
did you just put the minimal stuff here to get it to compile?  Because the actual struct has more, right?

Reply:
this is fine, but you should probably file a task to fill this out completely.  It's reasonable for third parties to want to use this and it's surprising that all of the numeric_limits definitions aren't there.

Reply:
[Filed](https://github.com/pytorch/pytorch/issues/22669).